src/Conda.jl

```diff
@@ -46,6 +46,11 @@ SCRIPTDIR
 "Prefix for the executable files installed with the packages"
 BINDIR
 
+"Returns the directory for python"
+@windows_only const PYTHONDIR = PREFIX
```

Comment:
Probably would be cleaner to wrote `const PYTHONDIR = @windows? PREFIX : BINDIR` ... that way, you wouldn't have to repeat the docstring.